src/lib/operators/jit_operator/operators/jit_aggregate.cpp

```diff
@@ -167,6 +167,22 @@ void JitAggregate::after_query(Table& out_table, JitRuntimeContext& context) con
   out_table.append_chunk(chunk_columns);
 }
 
+namespace {
+
+// The intermediary result of sum is always stored in a 64 bit data type
```

Comment:
Why?

Reply:
The behaviour in `Expression` is the same so that I copied this behaviour to ensure that the result tables have the same types. The decision how a query is processed should not have an effect on the results. I found this issue as some sqlite tests failed due to mismatching column definitions in the result tables.